source/query/skip.txt

```diff
@@ -0,0 +1,28 @@
+.. _query-bar-skip:
+
+==========================
+Skip a Number of Documents
+==========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+If the query bar has the :guilabel:`Skip` option, you can specify the
+first n-number of document to skip before returning the result set.
+
+To specify the number of documents to skip:
+
+#. Click :guilabel:`Options`.
+
+#. Enter the number to skip.
```

Comment:
See what I have as an example.

Reply:
I don't see an example on the page? I just see the code snippet `$skip: 5`, which is more like a syntactical example rather than a situational examples (like the ones present on other pages). I was envisioning something like:

"If you have a collection with 3,235 documents and you'd like to skip to the 435 document, you'd enter <number> in the "Skip" portion of the query bar". (or something like that)

Reply:
Agreed. Fixed.

Reply:
Damn that looks great! Right now it says:

 > $skip : 5

Should it be:

> $skip : 435

?